pkg/build/builder/cmd/scmauth/scmauth.go

```diff
@@ -0,0 +1,25 @@
+package scmauth
```

Comment:
Yeah, generally speaking this will be one of those things we'll externalize when abstracting scm access, IMHO.
@smarterclayton opinions?